### ISSUE TITLE
Changes to "Donate" button

### ISSUE THREAD

**Issue by maneesha** [carpentries/carpentries-hugo-theme#45]
See https://github.com/carpentries/new-website/issues/54 to consider a different color and location for the "Donate" button.

**Comment by maneesha** [comment]
This should be in Carpentries fire red (see https://info.carpentries.org/resources/communications/brand_identity.html).

**Comment by maneesha** [comment]
@OscarSiba  we can make the donate button be fire red, but the same accessibility concern would apply as referenced in https://github.com/carpentries/carpentries.org/issues/313

![Image](https://github.com/user-attachments/assets/a6574835-c91c-4a47-996a-d5a86fc677ae)

**Comment by OscarSiba** [comment]
Is it possible to make the whole button The Carpentries red (in the same way we made the other buttons blue on the blogs, etc.) and then the text "Donate" to be in white?

**Comment by maneesha** [comment]
Yes although accessibility of white on red is also a concern. See:
* https://coolors.co/contrast-checker/fff7f1-ff4955
* https://webaim.org/resources/contrastchecker/?fcolor=FFF7F1&bcolor=FF4955

**Comment by OscarSiba** [comment]
The contrast that works best and is still within our colour scheme seems to be the following:

Yellow background (#FFC700) and "Donate" in black text (#383838). 

Thanks @maneesha.

**Comment by maneesha** [comment]
@OscarSiba I am working on this change but realize I didn't ask enough questions.  

Right now, the "Donate" button is styled like this: 

![Image](https://github.com/user-attachments/assets/c55fb916-cac3-41fb-9150-f82386300fea)

When you mouse over, the style changes like this:

![Image](https://github.com/user-attachments/assets/5c1879fb-1127-4852-8e4d-db3805c86596)

Can you share what colors you'd like for each of the following:

* Original text
* Original background
* Mouse over text
* Mouse over background

Thanks!

**Comment by OscarSiba** [comment]
Hi Manessha. Ok. The way that the botton looks when you hover over it should be the way that is looks normally: 

![Image](https://github.com/user-attachments/assets/db2887c1-b3b5-4424-8ff9-28fab7d82fbd)

And then when you hover over it, it should change to the following: 

Yellow background (#FFC700) and "Donate" in black text (#383838).
